site/src/schema.ts

```diff
@@ -93,16 +93,24 @@ export const appMetadataSchema = {
                 "By default, the VS Code extension will assume that there's just a single application sitting at the root of the repo.",
             ].join('\n\n'),
         },
-        compatibleNcs: {
+        releases: {
             type: 'array',
             items: {
-                type: 'string'
+                type: 'object',
+                properties: {
```

Comment:
Would be nice to extend the schema with descriptions, after all externals are supposed to contribute to the JSON content.